shared/constants/platform.desktop.js

```diff
@@ -123,24 +123,31 @@ function findCacheRoot(): string {
 }
 
 function logDir(): string {
+  // See LogDir() functions in go/libkb/home.go.
+  //
+  // TODO: darwin and win32 cases are inconsistent with their LogDir()
```

Comment:
darwin just happens to work for prod mode, and so does win32 I'm guessing (do APPDATA and LOCALAPPDATA usually coincide?)